server/flyway/local_sql/sample_V1002__add_local_user_cognito_id.sql

```diff
@@ -42,12 +42,10 @@ INSERT INTO app_fam.fam_user (
     user_name,
     user_type_code,
     cognito_user_id,
+    user_guid,
     create_user
 )
 VALUES
-
-('JFERREIR','I','test-idir_278f48bd9mangledf7e618d@idir',CURRENT_USER),
-('NSAGLION','I','test-idir_3f1414bd1mangled18a1293c40@idir',CURRENT_USER),
-('CMENG','I','test-idir_a9888e8ac6a04mangled35df625bf@idir',CURRENT_USER),
-('OLIBERCH','I','test-idir_b7e191fa11dmangled9df1854b71@idir',CURRENT_USER);
-
+('CMENG','I','test-idir_a9888e8ac6a04mangled35df625bf@idir','A9888E8AC6A04MANGLED00135DF625BF',CURRENT_USER),
+('OLIBERCH','I','test-idir_b7e191fa11dmangled9df1854b71@idir','B7E191FA11MANLED818EC79DF1854B71',CURRENT_USER),
```

Comment:
Hi Ian, I think we need user_guid on local as well, cause now when we search IDIR or bceid user, it requires requester's user_guid, so I added placeholder for user_guid as well, thanks!

Reply:
Sure, good thought, thank you!! It does not affect me actually since I already adjusted it to my local copied.
Would you also adjust the comment in this script too for the user_guid? Thanks!!

Reply:
looks like it has the user_guid already? 
<img width="641" alt="image" src="https://github.com/user-attachments/assets/07b6f1da-b2e9-47f5-bb9d-dd5e5447ef43">

Reply:
um...I feel that's not very clear, so updated a bit, also added that we use upper cases for user_guid, do you mind review it again for me? Thanks so much Ian!!

Reply:
Oh, really?! Maybe it was meant cognito_id. Sorry for the confusion, I thought it did not have.